### PR TITLE
playbook: [club fair] first draft

### PR DESCRIPTION
**Pull request by jonleung**
**not ready for merge**

There are still typos and structural fixes that are required for this:

This pull request is work done for #86.

### CODE REVIEW

File: club_fair.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+### Overview
+
+### Checklist -->
+
+# Club fair/rush day
+
+**Outline:**
+
+- **[Overview](#overview)**
+  - [Be a fanatic peacock](#be-a-fanatic-peacock)
```
Comment (hellyeah) [inline club_fair.md]: If you don't think people think you're crazy, you're probably not being crazy enough.

Comment (zachlatta) [inline club_fair.md]: :+1:

On Mon, Aug 31, 2015 at 11:11 AM, David Fontenot <notifications@github.com>
wrote:

> In club_fair.md
> <https://github.com/hackedu/hackedu/pull/120#discussion_r38340737>:
>
> > +## First interactions matter
> > +
> > +# Getting up and running
> > +
> > +## Starting a new hackEDU club
> > +
> > +### Overview
> > +
> > +### Checklist -->
> > +
> > +# Club fair/rush day
> > +
> > +**Outline:**
> > +
> > +- **[Overview](#overview)**
> > +  - [Be a fanatic peacock](#be-a-fanatic-peacock)
>
> If you don't think people think you're crazy, you're probably not being
> crazy enough.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hackedu/pull/120/files#r38340737>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us

**Comment by zachlatta** [comment]
Going to go ahead and close this since we decided on doing the case studies format instead.

**Comment by jonleung** [comment]
@zachlatta, let's keep the branch though

**Comment by zachlatta** [comment]
Sounds good.

On Wed, Sep 9, 2015 at 9:19 AM, Jonathan Leung <notifications@github.com>
wrote:

> @zachlatta <https://github.com/zachlatta>, let's keep the branch though
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hackedu/pull/120#issuecomment-138961444>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us

**Comment by jonleung** [comment]
@zachlatta, I liked your suggestion [in this comment](https://github.com/hackedu/hackedu/pull/147#issuecomment-139003392)

I think I want to do the same for this pull request. Reopening this pull request.

**Comment by MaxWofford** [comment]
@jonleung Is this PR still active?

**Comment by zachlatta** [comment]
Closing due to inactivity.

**Pull request closed without merging**